iptables/iptables.go

```diff
@@ -109,7 +109,7 @@ func RunCmd(version, params string) error {
 
 // check if iptable chain alreay exists
 func ChainExists(version, tableName, chainName string) bool {
-	params := fmt.Sprintf("-t %s -L %s", tableName, chainName)
+	params := fmt.Sprintf("-t %s -nL %s", tableName, chainName)
```

Comment:
Is it worth documenting these flags here? I know that you can look them up from the manpage, but it might be helpful to put it in front of readers' faces.

Reply:
Thanks for suggestion. This is cherry-pick PR. will add it next time in the PR created against master